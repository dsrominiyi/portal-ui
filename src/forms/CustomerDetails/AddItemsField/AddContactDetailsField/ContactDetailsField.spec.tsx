import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import ContactDetailsField from '.';

let component: ShallowWrapper;
const onChange = jest.fn();

describe('ContactDetailsField', () => {
  test('ContactDetailsField matches snapshot', () => {
    component = shallow(<ContactDetailsField onChange={onChange} />);
    expect(component).toMatchSnapshot();
  });
});
