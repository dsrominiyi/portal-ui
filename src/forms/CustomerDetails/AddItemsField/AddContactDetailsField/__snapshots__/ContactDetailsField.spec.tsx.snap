// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactDetailsField ContactDetailsField matches snapshot 1`] = `
<AddItemsField
  errorText="Please add at least one method of contact"
  fieldText="Contact details"
  form={[Function]}
  itemName="contact"
  onChange={[MockFunction]}
  table={[Function]}
/>
`;
